src/MOIWrapper.jl

```diff
@@ -0,0 +1,356 @@
+export CbcOptimizer
+
+
+using MathOptInterface
+using Cbc.CbcCInterface
+
+const MOI = MathOptInterface
+const MOIU = MathOptInterface.Utilities
+const CbcCI = CbcCInterface
+
+mutable struct CbcOptimizer <: MOI.AbstractOptimizer
+    inner::CbcModel
+    # env
```

Comment:
Is this needed?

Reply:
@odow might be refering to the `# env` comment